Client/qtTeamTalk/channelstree.cpp

```diff
@@ -880,6 +880,7 @@ void ChannelsTree::slotUpdateTreeWidgetItem(QTreeWidgetItem* item)
             if (!item->data(COLUMN_MEDIAFILE, Qt::CheckStateRole).isNull())
                 item->setData(COLUMN_MEDIAFILE, Qt::CheckStateRole, QVariant());
         }
+        item->setData(COLUMN_ITEM, Qt::AccessibleTextRole, QString("%1: %2").arg(channame).arg((item->isExpanded()? tr("Expanded"):tr("Collapsed"))));

```

Comment:
Can we use Qt::AccessibleTextRole on classroom checkboxes for media stream/voice/webcam? You said these options were "invisible" to the screen-reader, so maybe this way we can announce the current state

Reply:
I don't reminder, where this checkboxes are?